app/views/stories/_form.html.erb

```diff
@@ -13,6 +13,11 @@
 
   <%= f.hidden_field :project_id, value: params[:project_id] %>
 
+  <div class="field story_status">
+    <%= f.label :status, "Status" %>
+    <%= f.select :status, options_for_select({ "Pending" => "pending", "Approved" => "approved", "Rejected" => "rejected" }, selected: @story.status), class: "project-story-approved" %>
+  </div>
```

Comment:
With the new status field, this seems a little bit moved, I expected the preview to be aligned with the description field at least as it was before.

The first thing I want to write is the story's title. can we move the status field after the extra info?

All of this is purely personal opinion.

![image](https://github.com/fastruby/points/assets/7331511/16494a59-82f9-4953-9b29-f36ff270a942)

Reply:
True. Will fix

Reply:
So, I moved the status selector, but I noticed that the description preview is actually in the right place, according to the [scss](https://github.com/fastruby/points/blob/main/app/assets/stylesheets/stories.scss#L95-L97) and the app version currently in production.